src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -853,8 +905,31 @@ int fx_muxer_t::read_config_and_execute(
         append_realpath(path, &probe_realpaths);
     }
 
+    // 'No candidate fx roll forward' is disabled by default. It can be enabled through:
+    // 1. Command line argument (--no-candidate-fx-roll-forward)
```

Comment:
I was thinking about the naming - to make it slightly more intuitive from readability perspective, what do you think about "RollForwardOnNoCandidateFX"?